<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>630</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Projectionist</string>
  </property>
  <property name="windowIcon">
   <iconset resource="splash.qrc">
    <normaloff>:/images/images/icon.png</normaloff>:/images/images/icon.png</iconset>
  </property>
  <property name="animated">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayoutMain">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>-1</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>630</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuTheme">
     <property name="title">
      <string>Theme</string>
     </property>
     <addaction name="actionLight"/>
     <addaction name="actionDark"/>
     <addaction name="actionAuto"/>
    </widget>
    <addaction name="menuTheme"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>48</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::BottomToolBarArea|Qt::TopToolBarArea</set>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionExit"/>
   <addaction name="actionTheme"/>
  </widget>
  <widget class="QToolBar" name="toolBarApps">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="baseSize">
    <size>
     <width>96</width>
     <height>0</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBarApps</string>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftToolBarArea|Qt::RightToolBarArea</set>
   </property>
   <property name="iconSize">
    <size>
     <width>96</width>
     <height>96</height>
    </size>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionBash"/>
   <addaction name="actionHTML"/>
   <addaction name="actionPHP"/>
   <addaction name="actionPython"/>
   <addaction name="actionArduino"/>
   <addaction name="actionKotlin"/>
  </widget>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="buttons.qrc">
     <normaloff>:/glass/buttons/glassButtonExit.png</normaloff>:/glass/buttons/glassButtonExit.png</iconset>
   </property>
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="toolTip">
    <string>Exit Projectionist</string>
   </property>
   <property name="statusTip">
    <string>Exit Projectionist</string>
   </property>
  </action>
  <action name="actionLight">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Light</string>
   </property>
  </action>
  <action name="actionDark">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Dark</string>
   </property>
  </action>
  <action name="actionAuto">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Auto</string>
   </property>
  </action>
  <action name="actionTheme">
   <property name="icon">
    <iconset resource="buttons.qrc">
     <normaloff>:/glass/buttons/glassButtonTheme.png</normaloff>:/glass/buttons/glassButtonTheme.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Theme</string>
   </property>
   <property name="toolTip">
    <string>Change Theme</string>
   </property>
   <property name="statusTip">
    <string>Change Application Theme</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="buttons.qrc">
     <normaloff>:/glass/buttons/glassButtonAbout.png</normaloff>:/glass/buttons/glassButtonAbout.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="icon">
    <iconset resource="buttons.qrc">
     <normaloff>:/glass/buttons/glassButtonQt.png</normaloff>:/glass/buttons/glassButtonQt.png</iconset>
   </property>
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionHTML">
   <property name="icon">
    <iconset resource="buttons.qrc">
     <normaloff>:/glass/buttons/glassButtonHtml.png</normaloff>:/glass/buttons/glassButtonHtml.png</iconset>
   </property>
   <property name="text">
    <string>HTML</string>
   </property>
   <property name="toolTip">
    <string>HTML Project</string>
   </property>
  </action>
  <action name="actionPHP">
   <property name="icon">
    <iconset resource="buttons.qrc">
     <normaloff>:/glass/buttons/glassButtonPhp.png</normaloff>:/glass/buttons/glassButtonPhp.png</iconset>
   </property>
   <property name="text">
    <string>PHP</string>
   </property>
   <property name="toolTip">
    <string>PHP Project</string>
   </property>
  </action>
  <action name="actionPython">
   <property name="icon">
    <iconset resource="buttons.qrc">
     <normaloff>:/glass/buttons/glassButtonPython.png</normaloff>:/glass/buttons/glassButtonPython.png</iconset>
   </property>
   <property name="text">
    <string>Python</string>
   </property>
   <property name="toolTip">
    <string>Pyhton Project</string>
   </property>
  </action>
  <action name="actionArduino">
   <property name="icon">
    <iconset resource="buttons.qrc">
     <normaloff>:/glass/buttons/glassButtonArduino.png</normaloff>:/glass/buttons/glassButtonArduino.png</iconset>
   </property>
   <property name="text">
    <string>Arduino</string>
   </property>
   <property name="toolTip">
    <string>Arduino Project</string>
   </property>
  </action>
  <action name="actionBash">
   <property name="icon">
    <iconset resource="buttons.qrc">
     <normaloff>:/glass/buttons/glassButtonBash.png</normaloff>:/glass/buttons/glassButtonBash.png</iconset>
   </property>
   <property name="text">
    <string>Bash</string>
   </property>
   <property name="toolTip">
    <string>Bash Project</string>
   </property>
  </action>
  <action name="actionKotlin">
   <property name="icon">
    <iconset resource="buttons.qrc">
     <normaloff>:/glass/buttons/glassButtonKotlin.png</normaloff>:/glass/buttons/glassButtonKotlin.png</iconset>
   </property>
   <property name="text">
    <string>Kotlin</string>
   </property>
   <property name="toolTip">
    <string>Kotlin Project</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="splash.qrc"/>
  <include location="buttons.qrc"/>
 </resources>
 <connections/>
</ui>
